www/tutorial/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "angular2-quickstart",
   "version": "1.0.0",
   "scripts": {
-    "start": "tsc && concurrently \"npm run tsc:w\" \"npm run lite\" ",
-    "lite": "lite-server",
-    "postinstall": "typings install",
-    "tsc": "tsc",
+    "build": "webpack",
+    "start": "webpack-dev-server  & open http://localhost:8080/",
+    "dev": "webpack-dev-server --inline --hot & open http://localhost:8080/",
+    "try-dev": "tsc && concurrently \"npm run tsc:w\" \"npm run lite\" ",
```

Comment:
actually, the script I added (try-dev) doesn't seem to be working.  i think the webpack one was especially not working before cause i had a port conflict -- now the webpack one is working with a refresh....it would be cool if it would just do that on its own, but i can live with it